tests/unit/testStationsFunction.unitTest.js

```diff
@@ -1,9 +1,32 @@
 const LambdaTester = require('lambda-tester')
 const GetTestStationsFunction = require('../../src/functions/getTestStationsLambda')
+const GetTestStationsEmailFunction = require('../../src/functions/getTestStationEmails')
+
 
 describe('getTestStations', () => {
   it('should return a promise', () => {
     return LambdaTester(GetTestStationsFunction)
       .expectResolve()
   })
 })
+
+describe('getTestStationsEmail', () => {
+  it('should return an error', () => {
```

Comment:
These tests descriptions could really use some more context e.g.
"should return an error when sending no parameters"

Reply:
Done